stubs/sympy-stubs/assumptions/assume.pyi

```diff
@@ -1,7 +1,7 @@
 from collections.abc import Generator
 from contextlib import contextmanager
 from typing import Any, Callable
-from typing_extensions import Self
+from typing_extensions import Never, Self
```

Comment:
Looks like this import isn't used?

Reply:
Yes that slipped through ty. I'm used to unused imports getting removed ^^"
```suggestion
from typing_extensions import Self
```